### ISSUE TITLE
RemovedInDjango110Warning

### ISSUE THREAD

**Issue by astronouth7303** [adamalton/django-csp-reports#10]
```
.../.env/src/django-csp-reports/cspreports/urls.py:7: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got report_csp). Pass the callable instead.
  url(r'^report/$', 'report_csp', name='report_csp'),

.../.env/src/django-csp-reports/cspreports/urls.py:7: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^report/$', 'report_csp', name='report_csp'),
```

**Comment by stucox** [comment]
@adamalton What versions of Django does this app support? If just 1.8+, could update this to the new syntax; for older versions, probably worth testing somehow & making it compatible with both before & after 1.10.

**Comment by adamalton** [comment]
Could one of you please do a quick check of #10 before I merge it.  That will fix this.

**Comment by adamalton** [comment]
This is now fixed by #11.
